miasm2/core/graph.py

```diff
@@ -149,3 +159,44 @@ def dot(self):
                                                 self.edge2str(a, b))
         out += "}"
         return out
+
+
+
+    def compute_dominators(self):
+        """Compute dominators of the graph"""
+        nodes = self.nodes()
+        heads = self.heads()
```

Comment:
Ok, if you prefer to reduce the CPU footprint over the memory one, it sounds good for me in this case.
I wanted to ensure it was not a mistake.